pkg/adapter/containers_remote.go

```diff
@@ -173,6 +173,32 @@ func (r *LocalRuntime) KillContainers(ctx context.Context, cli *cliconfig.KillVa
 	return ok, failures, nil
 }
 
+// RemoveContainer removes container(s) based on CLI inputs.
+func (r *LocalRuntime) RemoveContainers(ctx context.Context, cli *cliconfig.RmValues) ([]string, map[string]error, error) {
+	var (
+		ok       = []string{}
+		failures = map[string]error{}
+	)
+
+	ids, err := iopodman.GetContainersByContext().Call(r.Conn, cli.All, cli.Latest, cli.InputArgs)
+	if err != nil {
+		if errors.Cause(err) == libpod.ErrNoSuchCtr {
+			return ok, failures, nil
```

Comment:
In the non remote code, we do a removal here with the --force flag, can/should we do similar here?

Reply:
i think this comment is misplaced? hes just getting containers here; are you thinking of line 192?

Reply:
Might be a lack of tea on my part, but in  pkg/adapter/containers.go when GetContainersByContext is called at line 114 (https://github.com/containers/libpod/pull/2549/files#diff-7b5097b329b325dc0490ec58b133a9aeR114) if the error is nosuchcontainer, r.RemoveContainersFromStorage(cli.InputArgs) is called rather than returning.  Just wondering why the difference between the two.

Reply:
The remote and local paths have always been different.  @baude I just matched what was there.